cv/qc-cv.obo

```diff
@@ -2137,3 +2137,22 @@ property_value: has_optional_column: QC:4000249 ! PCA Dimension 3
 property_value: has_optional_column: QC:4000250 ! PCA Dimension 4
 property_value: has_optional_column: QC:4000251 ! PCA Dimension 5
 
+
+[Term]
+id: QC:4000268
+name: Number of peaks for at least 50% total ion current per spectrum
```

Comment:
is this the "**Minimal** number of peaks explaining at least 50% of the total ion current of a spectrum"